unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/VideoPlayer/VideoPlayerHandler.cs

```diff
@@ -38,18 +37,11 @@ public void OnComponentCreated(IParcelScene scene, IDCLEntity entity)
         {
             isRendererActive = !loadingScreen.visible.Get();
 
-            // We need to check if the user interacted with the application before playing the video,
-            // otherwise browsers won't play the video, ending up in a fake 'playing' state.
-            hadUserInteraction = Helpers.Utils.IsCursorLocked;
-
-            if (!hadUserInteraction)
-                Helpers.Utils.OnCursorLockChanged += OnCursorLockChanged;
```

Comment:
We won't be checking for user interaction now?
For AudioStream we now have the `tryPlay()` but not for videos, right?

Reply:
video already has it retry logic, waiting user's interaction was not needed...
quoting yourself...
https://github.com/decentraland/sdk/issues/653#issuecomment-1477836863